setuptools/py31compat.py

```diff
@@ -1,17 +1,3 @@
-__all__ = ['get_config_vars', 'get_path']
```

Comment:
Why drop `__all__` here?

Reply:
`get_config_vars` and `get_path` have been removed from py31compat.py because Python 2.7 and 3.4+ can directly `from sysconfig import get_config_vars, get_path`.

Reply:
Shouldn't we change this to `__all__ = []` then?

Reply:
I don't think it's necessary, if nothing is exposed.

The others don't. For example: https://github.com/pypa/setuptools/blob/master/setuptools/py36compat.py

But happy to add back if you prefer!

Reply:
I think it's probably better to be explicit, since if `__all__` is not specified, `from py31compat.py import *` will import everything, but if `__all__` is set to `[]` it will import nothing.

I think it also sends a clearer message that this module provides no public members.